### PR TITLE
OSDOCS-2622: release note for CoreDNS version bump

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2622

Preview: https://deploy-preview-36191--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-notable-technical-changes

Tagging Release Notes Tracker: https://github.com/openshift/openshift-docs/issues/33497

### CODE REVIEW

**Comment by bmcelvee** [comment]
@rfredette would you mind reviewing this PR, please? Thanks!

**Comment by rfredette** [comment]
lgtm 👍

**Comment by bmcelvee** [comment]
@jechen0648 would you mind reviewing this PR please? Thank you!

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -235,6 +235,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-9-coreDNS-version-update"]
+=== CoreDNS update to version 1.8.4
+
+In {product-title} {product-version} CoreDNS uses version 1.8.4, which includes bug fixes.
```
Comment (skrthomas) [inline release_notes/ocp-4-9-release-notes.adoc:242]: need comma after {product-version}

**Review by skrthomas — COMMENTED** [review]
One lil comment. /lgtm

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (bmcelvee) [reply release_notes/ocp-4-9-release-notes.adoc]: Adding it back. Thanks!

**Comment by bmcelvee** [comment]
@jechen0648 would you mind reviewing this release note PR please? Thank you!

**Comment by bmcelvee** [comment]
Rebased and fixed header levels.

**Pull request merged**